### ISSUE TITLE
Wildcards Not working properly in DAS query

### ISSUE THREAD

**Issue by vetens** [dmwm/das2go#39]
Dear Experts,
I am having an issue where wildcards do not seem to be working properly in DAS queries. For example when I use the following query:
dataset=/QCD_Pt-170to300_MuEnrichedPt5_TuneCP5_13TeV_pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v3/MINIAODSIM

I receive the proper dataset, however dataset=/QCD_Pt-170to300_MuEnrichedPt5_TuneCP5_13TeV_pythia8/*/* acts as though I am searching for a dataset with that literal name, rather than the wildcards.

**Comment by vkuznet** [comment]
@vetens to speficy wild-card you should use '*' symbol, e.g. `dataset=/ZMM*/*/*` which specify proper wild-card in different places. Otherwise there is not way to distinguish empty string from wild-card meaning. And, since dataset has three slashes, all parts should be properly displayed, e.g. if you do `dataset=/ZMM*//` it will not match anything since after second and third slash DAS will try to match to empty strings.

**Comment by vetens** [comment]
Yes, my apologies, I will post the strings again with proper formatting
``dataset=/QCD_Pt-170to300_MuEnrichedPt5_TuneCP5_13TeV_pythia8/RunIIAutumn18MiniAOD-102X_upgrade2018_realistic_v15-v3/MINIAODSIM`` is the first query, which indeed directs to the desired database, while i.e.
``dataset=/QCD_Pt-170to300_MuEnrichedPt5_TuneCP5_13TeV_pythia8/*/*``
is interpreted with the asterisks literally, rather than as wildcards. This may be the same issue as  this other posted recently. https://github.com/dmwm/das2go/issues/38#issuecomment-1064050365
Strangely enough it lists the second string as an actually existing dataset, but then attempting i.e. to access the file list provides a "no results found".

**Comment by Tommaso93** [comment]
#38
